test/masq.test.js

```diff
@@ -392,14 +331,6 @@ describe('Test login procedure', () => {
     await masq._disconnect()
     expect(masq.isLoggedIn()).toBe(true)
     expect(masq.isConnected()).toBe(false)
-
-    await masq.connectToMasq()
```

Comment:
We could keep connectToMasq at the end of the test to make sure we can connect after the disconnect when not logged in and not connected